tensorflow/lite/micro/tflite_bridge/micro_error_reporter.cc

```diff
@@ -13,7 +13,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
```

Comment:
the include guard in micro_error_reporter.h needs to be updated (I can't comment on that file so adding it here).

Reply:
Done.